externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/result/ResultSetUtil.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.flink.result;
+
+import org.apache.flink.table.api.DataTypes
+import org.apache.flink.table.api.ResultKind
+import org.apache.flink.table.catalog.Column
+import org.apache.flink.types.Row
+
+/** Util for flink result sets. */
```

Comment:
```suggestion
/** Utility class for constructing the ResultSet. */
```

Reply:
But this is a scala object, not a class. How about `Utility objetct`?  @SteNicholas

Reply:
@link3280, I have updated the suggestion.